.github/workflows/dotnet.yml

```diff
@@ -1,33 +1,29 @@
-name: Build, Test & Publish
+name: Build and deploy
 
 on:
-  push:
-    paths:
-    - 'src/**'
-    - '.github/workflows/**'
-    branches: [ master ]
-  pull_request:
-    paths:
-    - 'src/**'
-    branches: [ master ]
+  workflow_dispatch:
+    inputs:
+        imageVersion:
+          description: 'Docker Image Version (x.x.x)'     
+          required: true
 
 jobs:
   build:
 
     runs-on: ubuntu-latest
 
     steps:
+    - name: set env 'WS_NAME'
+      run: echo ::set-env name=WS_NAME::nager-date
     - uses: actions/checkout@v2
+    - name: Setup .NET Core 3.1	
+      uses: actions/setup-dotnet@v1	
+      with:	
+        dotnet-version: 3.1.x
     - name: Setup .NET 6.0
       uses: actions/setup-dotnet@v1
       with:
         dotnet-version: 6.0.x
-    - name: Change LicenseKey
-      if: github.event_name != 'pull_request'
-      working-directory: ./src
-      run: |
-        sed -i 's/Thank you for supporting open source projects/${{secrets.LICENSE_KEY}}/g' Nager.Date/DateSystem.cs
```

Comment:
`if (string.IsNullOrEmpty(LicenseKey) ||
                !LicenseKey.Equals("Thank you for supporting open source projects"))
            {
                throw new NoLicenseKeyException();
            }`

we need to replace

Reply:
ideally we replace
`public static string LicenseKey = null;`with `public static string LicenseKey = "myplant";`